TranslatorReasonerAPI.yaml

```diff
@@ -1,14 +1,26 @@
 openapi: 3.0.1
 info:
   description: OpenAPI for NCATS Biomedical Translator Reasoners
-  version: 1.0.0
+  version: INSERT-VERSION-OF-YOUR-SERVICE-HERE
   title: OpenAPI for NCATS Biomedical Translator Reasoners
   contact:
     email: edeutsch@systemsbiology.org
   license:
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0.html
   termsOfService: INSERT-URL-HERE
+  x-trapi:
+    version: 1.1.0
+  x-translator:
+    component: INSERT-EITHER-ARA-OR-KP-HERE
+    team: INSERT-TRANSLATOR-TEAM-NAME-HERE
+    biolink_model_version: INSERT-BIOLINK-MODEL-VERSION-HERE
```

Comment:
Change this to match translator_extension schema?

The translator_extension schema currently uses [biolink-version](https://github.com/NCATSTranslator/translator_extensions/blob/62755f92c82d6aafa2ae55c394434640ebb9a0dd/x-translator/smartapi_x-translator_schema.json#L52) as the property name, with acceptable values following the official biolink releases: so versions like '1.6.1', '1.6.0'...